### DISCUSSION TITLE
SAE 2.1.0 Wärmepumpe angelegt mit GPIO Zähler 26, erscheit aber nicht im Stadus warum ?

### DISCUSSION THREAD

**Discussion by DrMabuse100** [camueller/SmartApplianceEnabler#439] [Installation und Betrieb]
Hallo ich habe ein  Wärmepumpe angelegt mit GPIO Zähler Nr. 26.
Die erscheint aber nicht in der Stadus übersicht warum ?
Also ich messe die Leistung der WP. über einen S0 Bus der geht auf GPIO Nr.26.
Und wenn überschuß PV da ist wird die WP über einer FritzDECT mit 5 Grad Höhergefahren.
Ich möchte eigentlich nur die Tatsächliche Leistungsaufnahme im SHM 2.0 mit darstellen.
Oder muss ich dann auch die FritztDECt rausschmeisen und die Steuerung den SAE über eien GPIO ausgang überlassen ?

Danke Freundlicher Gruß

**Comment by camueller** [comment]
In der Status-Übersicht sind nur Geräte aufgeführt, für die ein Schalter konfiguriert wurde.

**Comment by DrMabuse100** [comment]
Hallo Dankeschön für den Hinweis, also heist das die Wäremepumpe muss als Schalet konfiguriert werden ?

Freundlicher Gruß

**Comment by camueller** [comment]
Wenn die Wärmepumpe nur im Sunny Portal angezeigt werden soll, braucht sie keinen Schalter und wir auch nicht in der Statusanzeige angezeigt. Dort werden nur Geräte angezeigt, die vom SAE geschaltet werden sollen. Bei einer Wärmepumpe ist das vermutlich nicht der Fall.

**Comment by DrMabuse100** [comment]
Hallo, so ich komme mit den S0 und gehe auf den GPIO 26 Pin37 und 39=(Masse) leitungslänge ca.10m. Interner Wiedersatnd steht im SAE auf pull down.

Nur kommen SHM die werte nicht richtig raus, einaml 0W obwohl der 3p Zähler auf den display 20W zeigt und dann auch mal 84W.
An was liegt das evtl. ?

Vielen vielen dank und ein Freundlicher Gruß
